tests/testsuite/config.rs

```diff
@@ -309,6 +308,13 @@ f1 = 1
     assert_match("", &output);
 }
 
+#[cargo_test]
+fn config_ambiguous_filename_symlink_doesnt_warn() {
+    config_ambiguous_filename_symlink_doesnt_warn_general(|| {
+        symlink_config_to_config_toml();
+    });
+}
```

Comment:
nit: personally i would rather duplicate the tests than do this.  This makes it harder to follow what the tests are doing and to evolve them over time.

Reply:
Gosh.  OK.

Reply:
Now done.